packages/fxa-auth-server/lib/server.js

```diff
@@ -267,6 +267,33 @@ async function create(log, error, config, routes, db, translator, statsd) {
       }
     }
     response.header('Timestamp', `${Math.floor(Date.now() / 1000)}`);
+
+    const sessionToken = request.auth && request.auth.credentials;
+    if (
+      request.headers['user-agent'] &&
+      sessionToken &&
+      sessionToken.setUserAgentInfo
+    ) {
+      const {
+        browser: uaBrowser,
+        browserVersion: uaBrowserVersion,
+        os: uaOS,
+        osVersion: uaOSVersion,
+        deviceType: uaDeviceType,
+        formFactor: uaFormFactor,
+      } = request.app.ua;
+      sessionToken.setUserAgentInfo({
+        uaBrowser,
+        uaBrowserVersion,
+        uaOS,
+        uaOSVersion,
+        uaDeviceType,
+        uaFormFactor,
+        lastAccessTime: Date.now(),
+      });
+      db.touchSessionToken(sessionToken, request.app.geo);
```

Comment:
It's not a mysql hit, but it does run a lua fn in redis. I like the idea of updating on any use of the token, but I'd like to understand the effect on redis before we do. Is there a route that is lowish volume that'd get us the desired effect for sync?

Reply:
> Is there a route that is lowish volume that'd get us the desired effect for sync?

Fair point, I think in hindsight it is probably better to start small with this change. I can confirm in the sync slack what this route might be.

Outside of syncing I think we can get pretty close to "last used" value by touching the session token when we hit the `/account/attached_client` route. IIUC that route is only used when we load up settings and display users devices and apps.